elements/pfe-clipboard/test/pfe-clipboard.spec.js

```diff
@@ -0,0 +1,192 @@
+import { expect, elementUpdated } from "@open-wc/testing/index-no-side-effects";
+import { spy } from "sinon";
+import { createFixture } from "../../../test/utils/create-fixture";
+import PfeClipboard from "../dist/pfe-clipboard.js";
+
+const hexToRgb = hex => {
+  const [, r, g, b] = /^#?([a-f\d]{2})([a-f\d]{2})([a-f\d]{2})/.exec(hex);
+    return [
+        parseInt(r, 16),
+        parseInt(g, 16),
+        parseInt(b, 16)
+      ];
+};
```

Comment:
Good helper script that could be moved to a utilites folder somewhere.

Reply:
It looks like it's duplicated in a lot of our tests.  Definitely a good candidate for a utils folder.

Reply:
Adding this as an issue: #1802